README.rst

```diff
@@ -87,6 +99,8 @@ ask-sdk                                |Standard Version| |Standard Downloads|
 ask-sdk-webservice-support             |Webservice Version| |Webservice Downloads|
 flask-ask-sdk                          |Flask Sdk Version| |Flask Sdk Downloads|
 django-ask-sdk                         |Django Sdk Version| |Django Sdk Downloads|
+ask-sdk-jinja-renderer                 |Jinja Renderer Sdk Version| |Jinja Renderer Sdk Downloads|
+ask-smapi-sdk                          |Smapi Sdk Version| |Smapi Sdk Downloads|
```

Comment:
Can we add a section in the `README` regarding SMAPI SDK? Just couple sentences could be good, and a link to the smapi sdk's readme will be fine.